.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+on: [release]
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.8]
+    
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+        
+    - name: Install pypa/build
+      run: python -m pip install setuptools build wheel --user
+
+    - name: Build a binary wheel
+      run: python setup.py sdist bdist_wheel
+        
+    - name: Publish distribution 📦 to Test PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        password: ${{ secrets.TEST_PYPI_API_TOKEN }}
```

Comment:
is there any reason to post to Test PyPI if we're going to automatically post to regular PyPI? In other words, will our test PyPI be helpful here?

Or, maybe we use Test PyPI differently (push periodically to test an auto-release trigger?

Reply:
The test system is used mostly to ensure that the publishing scripts work. Once those are solid, then yes, we could just publish regularly. Still, it's always chicken and the egg.

There's always lots of room for innovation here. :)

Reply:
got it!